docs/competitive-programming/top-view-of-BT.mdx

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
Just a suggestion please try to keep the width of each line in the file to be at most 79 characters. It makes the content center heavy and people won't have to scroll left or right to see what you have written.

Reply:
I don't get what you mean. Can you elaborate?

Reply:
Like each line should contain up to 79 characters only.

Reply:
Alright, I'll make sure that happens in the future.